Sources/YSwift/YSubscription.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import Yniffi
+
+/// Handler for an active subscription.
+/// Once the subscription is deinitialized, it will automatically unsubscribe.
+/// You can explicitly cancel the subscription by calling `cancel`.
+public final class YSubscription {
+    private var subscription: Yniffi.YSubscription?
+
+    init(subscription: Yniffi.YSubscription) {
+        self.subscription = subscription
+    }
+
+    public func cancel() {
```

Comment:
Felt that leaving a cancel method publicly available was useful when explicitly canceling, to avoid warnings in Swift when the variable was assigned but not read.